draft-gruessing-moq-requirements.md

```diff
@@ -99,6 +99,20 @@ and the goal is to provide a protocol stack that looks like this:
 
 Not all of the proposals for "Media Over QUIC" follow this model, but for the ones that do, it seems useful to have a name for "the protocol layer immediately beneath media".
 
+Within this document, we extend the latency requirement categories for streaming media described in {{I-D.draft-ietf-mops-streaming-opcons}}:
+
+- ultra low-latency (less than 1 second)
+- low-latency live (less than 10 seconds)
+- non-low-latency live (10 seconds to a few minutes)
+- on-demand (hours or more)
+
+These latency bands were appropriate for streaming media, which was the target for {{I-D.draft-ietf-mops-streaming-opcons}}, but some realtime media may have requirements that are significantly less than "ultra-low latency". Within this document, we are also using
+
+- ull500 (less than 500 ms)
+- ull100 (less than 100 ms)
```

Comment:
My only view on this bikeshed - I would propose we refer to these as "Sub-500" and "Sub-100" just for readability.